pyqg/layered_model.py

```diff
@@ -275,48 +275,75 @@ def _initialize_model_diagnostics(self):
         self.add_diagnostic('entspec',
                 description='barotropic enstrophy spectrum',
                 function= (lambda self:
-                    np.abs((self.Hi[:,np.newaxis,np.newaxis]*self.qh).sum(axis=0))**2/self.H) )
+                    np.abs((self.Hi[:,np.newaxis,np.newaxis]*self.qh).sum(axis=0))**2/self.H),
+                units='meters squared per seconds squared',
```

Comment:
For units, I recommend we use [UDUNITs conventions](https://www.unidata.ucar.edu/software/udunits/udunits-2.2.28/udunits2.html). They are not very well documented unfortunately. But I think it's something like this

```suggestion
                units='m2 s-2',
```

Reply:
And the same for all units.

Reply:
I took a look at the UDUNITS documentation and they have exponents designated with `^` and the denominator with `/`, so this example would be `m^2/s^2` if I'm not mistaken.